common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -632,6 +635,25 @@ def get_problem_html(self, encapsulate=True):
             'weight': self.weight,
         }
 
+        # Handle demand hints
+        demand_hints = self.lcp.tree.xpath("//problem/demandhint/hint")
+        show_hint_button = len(demand_hints) > 0
+        demand_hint = None  # if set to something, will show up in UI
+        if hint_index is None:
+            hint_index = 0  # bootstrap case, the client will start at 0
+        elif len(demand_hints) > 0:
+            _ = self.runtime.service(self, "i18n").ugettext  # pylint: disable=redefined-outer-name
+            hint_element = demand_hints[hint_index]
+            if len(demand_hints) == 1:
+                prefix = _('Hint: ')
+            else:
+                # Translators: e.g. "Hint 1 of 3" meaning we are showing the first of three hints.
+                # Do not translate hint_num and hints_count
+                prefix = _('Hint ({hint_num} of {hints_count}): ').format(hint_num=hint_index + 1,
```

Comment:
OK, that makes sense. Well, the translators are used to `{word}` being a variable so you don't need to add the second sentence. But I would add one that says, `hint_count will always be greater than or equal to 2` (hint_count makes a bit more sense than "hints_count" but w/e) Thanks for the clarification